mender/values.yaml

```diff
@@ -316,6 +316,7 @@ gui:
     annotations: {}
     type: ClusterIP
     port: 80
+    portError: ""
```

Comment:
should this be `8080` by default?

Reply:
As of now it's disabled on the Helm chart, so by setting it it will be enabled starting from the next release. 
I guess it's fine and don't break anything, tough, so it's safe to set it

Reply:
ah sorry, only now I realize that this was for internal usage by the error middleware only (and traefik would return these errors through the regular port already), so is not really meant to be used by the outside world... hmm then I'm not sure if a different setting will have an effect there, but aligning it with the HM instance should be good in any case 👍...